image_view/src/nodes/video_recorder.cpp

```diff
@@ -108,6 +114,7 @@ int main(int argc, char** argv)
     bool stamped_filename;
     local_nh.param("stamped_filename", stamped_filename, false);
     local_nh.param("fps", fps, 15);
+    local_nh.param("tolerance", tolerance, 15.0);
```

Comment:
Would you mind making this a value from 0-1 instead? Additionally, I'd prefer something more descriptive to indicate that it's a tolerance for the FPS rate (e.g. `fps_tolerance_pct`).

Reply:
I agree with switching it to 0-1. Then the "tolerance" can be interpreted as a fraction of the frame duration (1/fps). In that case, can I call the parameter something like `fps_tolerance_fraction` or maybe `fps_variation_allowance`?